src/test/suite/completion.test.ts

```diff
@@ -44,13 +44,15 @@ import {
   setupForCompletionsTests,
   mockGetDebounceConfig,
 } from "./completion.driver";
+import { SuggestionShownRequestMatcher } from "./utils/SuggestionShownRequestMatcher";
 
 describe("Should do completion", () => {
   const SPACES_INDENTATION = "    ";
   const TAB_INDENTATION = "\t";
-  const WAIT_LONGER_THAT_DEBOUNCE = 250;
+  const WAIT_LONGER_THAN_SHORT_DEBOUNCE = 350;
```

Comment:
remove `WAIT` from const name, its not a command, its a const (same for line 55)